test/cppsuite/src/main/test.cpp

```diff
@@ -168,11 +168,8 @@ test::run()
         it->finish();
 
     /* Validation stage. */
-    if (_operation_tracker->enabled()) {
-        std::unique_ptr<configuration> tracking_config(_config->get_subconfig(OPERATION_TRACKER));
-        this->validate(_operation_tracker->get_operation_table_name(),
-          _operation_tracker->get_schema_table_name(), _workload_manager->get_database());
-    }
+    this->validate(_operation_tracker->enabled(), _operation_tracker->get_operation_table_name(),
```

Comment:
We now always call` validate`. If one overrides this function, they can validate their test without having the tracking component enabled.